README.md

```diff
@@ -5,12 +5,19 @@
 
 
 > Module 1, Week 5 - 1st Capstone Project
-> Continuing work for Microverse Program - Week 5
+> Continuing work for Microverse Program - Capstone Project for Module 1 on Microverse Program - Week 5
 
 
-![screenshot](./app_screenshot.png) <!--- Place a screenshot of my project here -->
+# Images & Video
+
+## Images
+
+![screenshot](./app-screenshot.jpg)
+
+## Video
+
+https://www.loom.com/share/31b53550cc1d4161bcee5abc6a1872a8
```

Comment:
- [ ] : Video is great, the app looks like woring as expected, but sadly I have to follow the requirements provided by microverse:

![image](https://user-images.githubusercontent.com/77038610/152653403-16f178e1-008f-4d50-84ce-ea4888b0c5cb.png)
You are supposed to be explaining your project while presenting it, and talk briefly about it a feature that you are proud of. 
You still have around 10 hours to submit projects, so please dont worry about the deadline, I recommend you to work on video for 10 min and submit again!